changelog/1.16.0.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "1.16.0"
+description: "Released on 02/22/2021"
+---
+
+### Breaking Changes ❗
+
+- The deprecated `Services` feature is now **removed**. All resources created
+  for Services are preserved for data recovery purposes (volumes,
+  database tables, etc.), but are not accessible through the Coder platform.
+  Please reference
+  [Docker in Environments](../environments/cvms) for
+  a guide on the new workflow for running containers with your Environment.
```

Comment:
Do we have a guide explaining how to recover this data?

Reply:
seems out of scope